junqo_back/src/main.ts

```
@@ -1,8 +1,47 @@
+      whitelist: true,
+      transform: true,
+      forbidNonWhitelisted: true,
+      enableDebugMessages: process.env.NODE_ENV === 'production' ? false : true,
+      transformOptions: {
+        enableImplicitConversion: true,
+      },
+    }),
+  );
+  const port = Number(process.env.BACK_PORT ?? 3000);
+
+  if (isNaN(port) || port < PORT_MIN || port > PORT_MAX) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)